package.json

```diff
@@ -26,7 +26,8 @@
         "react-map-gl": "^4.0.5",
         "sass": "^1.15.2",
         "sass-loader": "^7.1.0",
-        "viewport-mercator-project": "^6.1.0"
+        "viewport-mercator-project": "^6.1.0",
+        "react-dropdown-tree-select": "^1.20.1"
```

Comment:
so i am going to remove this from this commit, since the focus is on the other meetings